.gitignore

```diff
@@ -1,2 +1,3 @@
 __pycache__
-.vscode/
\ No newline at end of file
+.vscode/
```

Comment:
comment not solved:
https://github.com/keyboard-slayer/poc-joda-odoo/pull/1#discussion_r829111551